Jenkinsfile

```diff
@@ -119,7 +127,10 @@ stage("Check dependent projects against updated schema") {
           [$class: 'StringParameterValue',
             name: 'SCHEMA_BRANCH',
             value: env.BRANCH_NAME],
-        ]
+          [$class: 'StringParameterValue',
+            name: 'SCHEMA_COMMIT',
+            value: env.GIT_COMMIT]
+        ], wait: wait
```

Comment:
Yeah, I think we could do something like that. Probably best if we remove the timers in this PR to avoid confusion.